mmdet/core/evaluation/coco_utils.py

```diff
@@ -1,12 +1,19 @@
+import itertools
+
 import mmcv
 import numpy as np
 from pycocotools.coco import COCO
 from pycocotools.cocoeval import COCOeval
+from tabulate import tabulate
```

Comment:
Use `terminaltables` instead of `tabulate` since we have already used it in other methods.